documentation/src/main/asciidoc/user_guide/chapter-66-Map_Reduce_framework.adoc

```diff
@@ -2,6 +2,10 @@
 
 MapReduce is a programming model allowing transparent distributed processing of very large data sets over data grids. The name MapReduce comes from an idea of using two distinct computational phases of map and reduce. In the map phase, master node that initiates a task takes the task input, divides it and sends tasks for map phase execution on the grid. Each node in turns executes a map function on its input returning intermediate results back to master node. Master node task collects all intermediate results from map phase combines them by intermediate result keys and sends intermediate keys/values for reduction on the grid. Finally master tasks node receives all results from reduction phases and returns the final result to invoker of the MapReduce task.
 
+CAUTION: The Map/Reduce framework has been deprecated and will be removed with
```

Comment:
Ah I wasn't sure if we put deprecations there, but from [1] I guess we are.  In that case I think we are missing a ton of stuff!

http://infinispan.org/docs/8.2.x/upgrading/upgrading.html#_spring_3_support_is_deprecated